### PR TITLE
Remove extra directories

### PR DESCRIPTION
**Pull request by ZitaNemeckova**
Removes directories that are created by default and we don't need them at all.
Deletes models, mailers and jobs.

@AparnaKarve can you check it out, please. Thanks :)

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -13,21 +13,3 @@ RDoc::Task.new(:rdoc) do |rdoc|
...
-APP_RAKEFILE = File.expand_path('../test/migration/Rakefile', __FILE__)
-load 'rails/tasks/engine.rake'
-
-load 'rails/tasks/statistics.rake'
-
-require 'bundler/gem_tasks'
-
-require 'rake/testtask'
-
-Rake::TestTask.new(:test) do |t|
```
Comment (ohadlevy) [inline Rakefile]: do we really want to remove the testing framework for rails? (not sure if we wont end some ruby code at some point?)

Comment (ZitaNemeckova) [reply Rakefile]: I would prefer to add it if we need it than keep it just in case.

Comment (ZitaNemeckova) [reply Rakefile]: And it doesn't work as is.
```
rake aborted!
LoadError: cannot load such file -- miq_v2v_ui_plugin
/Users/zita/Desktop/ManageIQ/miq_v2v_ui_plugin/test/migration/config/application.rb:6:in `require'
/Users/zita/Desktop/ManageIQ/miq_v2v_ui_plugin/test/migration/config/application.rb:6:in `<top (required)>'
/Users/zita/Desktop/ManageIQ/miq_v2v_ui_plugin/test/migration/Rakefile:4:in `require_relative'
/Users/zita/Desktop/ManageIQ/miq_v2v_ui_plugin/test/migration/Rakefile:4:in `<top (required)>'
/usr/local/lib/ruby/gems/2.3.0/gems/railties-5.0.6/lib/rails/tasks/engine.rake:3:in `load'
/usr/local/lib/ruby/gems/2.3.0/gems/railties-5.0.6/lib/rails/tasks/engine.rake:3:in `block (2 levels) in <top (required)>'
/usr/local/lib/ruby/gems/2.3.0/gems/railties-5.0.6/lib/rails/tasks/engine.rake:2:in `block in <top (required)>'
/usr/local/lib/ruby/gems/2.3.0/gems/railties-5.0.6/lib/rails/tasks/engine.rake:72:in `<top (required)>'
/Users/zita/Desktop/ManageIQ/miq_v2v_ui_plugin/Rakefile:18:in `load'
/Users/zita/Desktop/ManageIQ/miq_v2v_ui_plugin/Rakefile:18:in `<top (required)>'
/usr/local/lib/ruby/gems/2.3.0/gems/rake-11.3.0/exe/rake:27:in `<top (required)>'
Tasks: TOP => load_app
(See full trace by running task with --trace)
```

**Comment by martinpovolny** [comment]
@ohadlevy, @priley86 : what is the status of this PR?

What is the status of moving this repo under ManageIQ?

I am seeing with growing concerns that changes to support V2V are to be backported to Gaprindashvili while elementary stuff is not yet resolved here.

**Comment by priley86** [comment]
@martinpovolny - will try to resolve this today and get back w/ you...need to chat w/ @AparnaKarve

**Comment by priley86** [comment]
i have synchronized alpha branch w/ master (so master now has latest). Merged #69 . Merging this. We now have latest code ready in master for w/e is needed there. 

I would hope that we have more time to iterate on the currently open issues before releasing v2v.

**Comment by priley86** [comment]
let me know if you have any more needs here @martinpovolny @AparnaKarve .

**Pull request merged**